app/scripts/metamask-controller.js

```diff
@@ -1498,7 +1499,7 @@ export default class MetamaskController extends EventEmitter {
           category: MetaMetricsEventCategory.PushNotifications,
           properties: {
             notification_type: notification.type,
-            chain_id: notification?.chain_id,
+            chain_id: hexToDecimal(notification?.chain_id),
```

Comment:
You can skip these. Our notification service returns decimal chain_id's.
CC @MetaMask/notifications